data/mupen64plus.ini

```diff
@@ -16,15 +16,17 @@ CountPerOp=1
 [34AB1DEA3111A233A8B5C5679DE22E83]
 GoodName=007 - The World is Not Enough (E) (M3) [!]
 CRC=3B941695 F90A5EEB
-SaveType=Controller Pack
+SaveType=None
```

Comment:
Why change all these to "None"? There is a CoreGetRomSettings function that will return this information

Reply:
These are for games that only support Mempak, no in cart saves. But to be honest, I did a mass replace with regex, and I'm currently fixing these on a per-game basis. But it's a time consuming process...